www/less/misc.less

```diff
@@ -53,6 +53,29 @@ a{
     padding-bottom:0px;
 }
 
+// Horizontal divider between title/description and lead section text
+#content_block_0_hr {
+    border:1px solid #999;
```

Comment:
transform:scaleY is not necessary. Since we're iOS8 and above we can use media selectors `@media (-webkit-min-device-pixel-ratio: 2) {` to adjust the border width to 0.5 for retina screens.